### ISSUE TITLE
Performance SDK: "Unsupported use of UIKit API off the main thread" errors

### ISSUE THREAD

**Issue by alexshikov** [xamarin/GoogleApisForiOSComponents#347]
Using `Xamarin.Firebase.iOS.PerformanceMonitoring (3.1.1)` nuget package traces lots of errors in the console:

```
2019-12-11 22:15:46.218540+0200 XXX.iOS[60350:546777] [Assert] Unsupported use of UIKit API off the main thread: UIAccessibilityIsAssistiveTouchRunning()
2019-12-11 22:15:46.218728+0200 XXX.iOS[60350:546777] [Assert] Unsupported use of UIKit API off the main thread: UIAccessibilityIsGuidedAccessEnabled()
2019-12-11 22:16:14.969980+0200 XXX.iOS[60350:546764] [Assert] Unsupported use of UIKit API off the main thread: UIAccessibilityIsAssistiveTouchRunning()
2019-12-11 22:16:14.970575+0200 XXX.iOS[60350:546764] [Assert] Unsupported use of UIKit API off the main thread: UIAccessibilityIsGuidedAccessEnabled()
2019-12-11 22:16:14.971973+0200 XXX.iOS[60350:546764] [Assert] Unsupported use of UIKit API off the main thread: UIAccessibilityIsAssistiveTouchRunning()
2019-12-11 22:16:14.972305+0200 XXX.iOS[60350:546764] [Assert] Unsupported use of UIKit API off the main thread: UIAccessibilityIsGuidedAccessEnabled()
2019-12-11 22:16:14.972997+0200 XXX.iOS[60350:546764] [Assert] Unsupported use of UIKit API off the main thread: UIAccessibilityIsAssistiveTouchRunning()
2019-12-11 22:16:14.973139+0200 XXX.iOS[60350:546764] [Assert] Unsupported use of UIKit API off the main thread: UIAccessibilityIsGuidedAccessEnabled()
2019-12-11 22:16:14.973549+0200 XXX.iOS[60350:546764] [Assert] Unsupported use of UIKit API off the main thread: UIAccessibilityIsAssistiveTouchRunning()
```

The exact issue has been reported in the original library: https://github.com/firebase/firebase-ios-sdk/issues/4156 and seems it has been fixed recently.

**Comment by alexshikov** [comment]
As we here. Are there any plans to update the global firebase version from `6.5.0` to something newer? According to [the official release notes](https://firebase.google.com/support/release-notes/ios) there are at least 4 crashes fixed in [6.13.0 - November 14, 2019](https://firebase.google.com/support/release-notes/ios#performance-monitoring) and another pretty awful crash (95% of iOS 13 users) introduced in [6.10.0 - October 8, 2019](https://firebase.google.com/support/release-notes/ios#performance-monitoring_2)

**Comment by SotoiGhost** [comment]
Hi @alexshikov! Letting you know that Firebase Performance Monitoring v3.1.9 (a.k.a 6.15.0) has been released! Hope this helps you! If something related to this happens, please, let us know. Best!
